cfme/tests/cloud/test_keypairs.py

```diff
@@ -34,6 +30,10 @@ def test_keypair_crud(a_provider):
         * Select Cloud Provider.
         * Also delete it.
     """
+    tag = ('Department', 'Accounting')
     keypair = KeyPair(name=fauxfactory.gen_alphanumeric(), provider=a_provider)
     keypair.create()
+    keypair.add_tag(tag)
```

Comment:
**REQUIRED**
I wouldn't consider tagging part of CRUD, this should be in a separate test function.  A complete test would assert against the tags after they've been added to the KeyPair